lib/pure/strutils.nim

```diff
@@ -2369,6 +2369,10 @@ proc formatBiggestFloat*(f: BiggestFloat, format: FloatFormatMode = ffDefault,
       # but nothing else is possible:
       if buf[i] in {'.', ','}: result[i] = decimalSep
       else: result[i] = buf[i]
+      if result[^1] == decimalSep:
```

Comment:
```nim

when (NimMajor, NimMinor) >= (1, 1):
  if ...
```

Reply:
1. This would be then needed for every changed test too.
2. If this won't be backported (and it shouldn't be, IMO), does this switch do anything useful? (That part of the code will only exist in Nim versions >= 1.1)

Reply:
The tests are fine the way they are. This switch does *emulate* 1.0.x, so IMO we should strive to make it work.

Reply:
Done.